docs/source/using/managing-minishift.adoc

```diff
@@ -53,6 +53,43 @@ the GitHub issue https://github.com/minishift/minishift/issues/179[#179].
 The link:../command-ref/minishift_delete{outfilesuffix}[`minishift delete`] command deletes the OpenShift cluster,
 and also shuts down and deletes the Minishift VM. No data or state are preserved.
 
+[[choosing-iso-image]]
+== Choosing the ISO image
+
+When you start Minishift, it downloads a live ISO image that the hypervisor uses to
+provision the host VM.
```

Comment:
s/provision the host VM/provision the minishift VM/ or s/provision the host VM/provision the VM/. `host VM` seems incorrect to me.